crates/bevy_winit/src/accessibility.rs

```diff
@@ -140,9 +140,9 @@ fn update_accessibility_nodes(
 }
 
 /// Implements winit-specific `AccessKit` functionality.
-pub struct AccessibilityPlugin;
+pub struct WinitAccessibilityPlugin;
```

Comment:
After tinkering a bit with this plugin, I found that it essentially provides winit with an AccessKit adapter. I think this can be renamed to `AccessKitPlugin`.

Reply:
Sure!